Webmin/input/auxiliary/transition_notes.xml

```diff
@@ -0,0 +1,1700 @@
+sysctl_net_ipv4_conf_default_accept_redirects rule.
+</note>
+
+<note ref="22417" auth="KS">
+Check does exist in the RHEL6 prose, it can be automated and OVAL for it does 
+exist.
+rule=sysctl_net_ipv4_conf_all_send_redirects manual=no
+This check is split in the RHEL6 prose into the above and the 
+sysctl_net_ipv4_conf_default_send_redirects rule.
+</note>
+
+<note ref="22418" auth="KS">
+Partial check does exist in the RHEL6 prose, it can be automated and partial 
+OVAL for it does exist.
+rule=sysctl_net_ipv4_conf_all_log_martians manual=no
+This check is split in the RHEL6 prose into the above but no equivalent rule 
+exists for "default."
+</note>
+
+<note ref="22419" auth="KS">
+Check does exist in the RHEL6 prose, it can be automated and OVAL for it does 
+exist.
+rule=sysctl_net_ipv4_tcp_syncookies manual=no
+</note>
+
+<note ref="22429" auth="KS">
+Check does exist in the RHEL6 prose, it can be automated and OVAL for it does 
+exist.
+rule=service_rpcbind_disabled manual=no
+</note>
+
+<note ref="22431,22432,22433,22434" auth="KS">
+Check does exist in the RHEL6 prose, it can be automated and OVAL for it does 
+exist.
+rule=uninstall_rsh-server manual=no
+</note>
+
+<note ref="22456,22461,22462,22463,22474" auth="KS">
+Check does not exist in the RHEL6 prose, it can be automated and OVAL for it 
+does not exist.
+rule=null manual=no
+No check exists for the client side.
+</note>
+
+<note ref="22457" auth="KS">
+Check does not exist in the RHEL6 prose, it cannot be automated and OVAL for 
+it does not exist.
+rule=null manual=yes
+No automated way to determine the management interface.
+</note>
+
+<note ref="22458,22459" auth="KS">
+Check does exist in the RHEL6 prose, it can be automated and OVAL for it does 
+exist.
+rule=sshd_use_approved_ciphers manual=no
+V-22458 and V-22459 are essentially the same.
+</note>
+
+<note ref="22470" auth="KS">
+Partial check does exist in the RHEL6 prose, it can be automated and OVAL for it does 
+exist.
+rule=sshd_limit_user_access manual=no
+Prose focuses on blacklisting where we should prefer a whitelist.
+</note>
+
+<note ref="22489" auth="KS">
+Check does exist in the RHEL6 prose, it can be automated and OVAL for it does 
+exist.
+rule=sshd_enable_warning_banner manual=no
+</note>
+
+<note ref="780" auth="JB">
+This is incoherent. There is no apparent relationship between the title, description, and
+fixtext/checktext. The checking text lists user IDs in
+/etc/passwd, which have nothing to do with group IDs.  If something coherent is
+presented it could be considered for inclusion for RHEL 6, but even then, the
+possibilities suggest checking for a highly unlikely configuration which is not
+worth inclusion in a baseline.
+</note>
+
+<note ref="777" auth="JB">
+This is redundant to another rule requiring that the path be the vendor default.
+</note>
+
+<note ref="776,812,761,781,782,813,4382,11975" auth="DS">
+This is covered in the RHEL6 content.
+</note>
+
+</notegroup>
```

Comment:
this file should be dropped. it was used to document which RHEL5 STIG rules were going to be inherited into RHEL6 content.